src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div>
+        <a href="#logo">
+          <img 
+            class="logo"
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+      </div>
+      <nav class="nav_section">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="#apple">APPLE</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#samsung">SAMSUNG</a>
```

Comment:
```suggestion
            <a class="nav_link" href="#samsung">Samsung</a>
```
Тут текст пиши обычным кеглем, а черезе css делай аппер

Reply:
Исправил